### ISSUE TITLE
Supported platforms for embedded devices

### ISSUE THREAD

**Issue by Wvirgil123** [lvgl/lv_binding_js#45]
Hello, I would like to ask,
1. What is the real device shown in the readme?
2. What are the limitations or difficulties of porting lvgljs to embedded devices (such as esp32)? Is there a minimum performance or resource specification?

**Comment by derekstavis** [comment]
1. It's a Linux device
2. It would be difficult due to the JS runtime depending on libuv for networking and filesystem. It can be done but not an easy task.

**Comment by bnolan** [comment]
See #46

**Comment by 19chang19** [comment]
@derekstavis Hello, could you please tell me：
1. which SOC is on your Linux device, 
2. how much RAM it consumes, 
3. and how much storage the entire image ROM occupies (excluding pictures resources)? Or alternatively, how much storage space do the JS engine and LVGL library occupy?

**Comment by matthewruzzi** [comment]
Would it be possible to port this to run in something like JerryScript so it could be used on low power microcontrollers?

**Comment by HiweH** [comment]
> Would it be possible to port this to run in something like JerryScript so it could be used on low power microcontrollers?

yes,u can use JerryScript to replace the quickjs to get your work.

**Comment by matthewruzzi** [comment]
Has anyone created code examples for this?
